raiden/api/rest.py

```diff
@@ -272,6 +274,8 @@ def deposit(self, token_address, partner_address, amount):
             )
         except EthNodeCommunicationError as e:
             return make_response(str(e), httplib.REQUEST_TIMEOUT)
+        except InsufficientFunds as e:
```

Comment:
Ha, I've just did the test and then discarded it upon discovering that it is mocked and I can't test it properly.